README.md

```diff
@@ -29,10 +29,10 @@ Company                                           | Business/products
 [Cloudbeds](https://www.cloudbeds.com/)           | Hospitality Software                     | Web, DevOps                                    | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.cloudbeds.com/careers)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Cloudbeds-EI_IE1399374.11,20.htm)
 [Cloudbees](https://www.cloudbees.com/)           | CI/CD, Jenkins, Codeship                 | Java, Web, Go, Vuejs, Rails                    | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.cloudbees.com/careers)<br>:door: [Glassdoor](https://www.glassdoor.ca/Overview/Working-at-CloudBees-EI_IE1106279.11,20.htm)
-[Codelathe](https://www.getfilecloud.com/)        | Enterprise File Sharing, Sync and Backup | PHP, Web, Node, C++                            | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.getfilecloud.com/jobs)
+[Codelathe](https://www.getfilecloud.com/)        | Enterprise File Sharing, Sync and Backup | PHP, Web, Node, C++                            | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.getfilecloud.com/jobs)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-FileCloud-EI_IE2893485.11,20.htm) 
 [ConsenSys](https://consensys.net/)               | Blockchain studio                        | Many (see job page)                            | ?                                    | Anywhere                | :keyboard: [Jobs](https://consensys.net/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com/Reviews/ConsenSys-Reviews-E1613426.htm)
 [Crossover](https://www.crossover.com/)           | Full-time long-term projects             | Many (see job page)                            | ✗                                    | Anywhere                | :keyboard: [Jobs](https://app.crossover.com/x/marketplace/available-jobs)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Crossover-for-Work-EI_IE1041838.11,29.htm)
-[DataDog](http://www.datadoghq.com)               | Monitoring & analytics                   | Scala, Java, Python, Go                        | ✓                                    | Anywhere                | :keyboard: [Jobs](https://www.datadoghq.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Datadog-EI_IE762009.11,18.htm)
+[DataDog](http://www.datadoghq.com)               | Monitoring & analytics                   | Scala, Java, Python, Go                        | ✓                                    | US & EU                 | :keyboard: [Jobs](https://www.datadoghq.com/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Datadog-EI_IE762009.11,18.htm)
```

Comment:
What is this change based on? Looks like they have positions globally, like [this one](https://www.datadoghq.com/careers/detail/?gh_jid=1825971).

Reply:
Right, my mistake. I checked several jobs and the choice was only EU\US, so I thought they don't have anything else.